node/subsystem-types/src/messages.rs

```diff
@@ -789,6 +789,33 @@ pub enum ApprovalCheckError {
 	Internal(Hash, CandidateHash),
 }
 
+
+/// Describes a relay-chain block by the para-chain candidates
```

Comment:
is this the new standard way of writing relay chain and parachain?

Reply:
No. I'll address this.